src/api/api_pass.cc

```diff
@@ -130,6 +130,7 @@ REGISTER_PASS(RewriteUnsafeSelect);
 REGISTER_PASS(Inline);
 REGISTER_PASS(IRTransform);
 REGISTER_PASS(VectorizeLoop);
+REGISTER_PASS(SkipVectorize);
```

Comment:
I missed adding this line in https://github.com/dmlc/tvm/pull/3222, so I added it here.